pkg/api/server/register_containers.go

```diff
@@ -24,44 +22,63 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//    description: container name
 	// responses:
 	//   '201':
-	//    description: tbd
+	//       $ref: "#/responses/CtrCreateResponse"
 	//   '400':
 	//       "$ref": "#/responses/BadParamError"
 	//   '404':
 	//       "$ref": "#/responses/NoSuchContainer"
 	//   '409':
-	//       "$ref": "#/types/ConflictError"
+	//       "$ref": "#/responses/ConflictError"
 	//   '500':
-	//      "$ref": "#/types/InternalError"
+	//      "$ref": "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/containers/create"), APIHandler(s.Context, generic.CreateContainer)).Methods(http.MethodPost)
 	// swagger:operation GET /containers/json containers listContainers
-	//
-	// List containers
-	//
 	// ---
+	// summary: List containers
+	// description: Returns a list of containers
+	// parameters:
+	//  - in: query
+	//    name: filters
+	//    type: string
+	//    description: |
+	//       Returns a list of containers.
+	//        - ancestor=(<image-name>[:<tag>], <image id>, or <image@digest>)
+	//        - before=(<container id> or <container name>)
+	//        - expose=(<port>[/<proto>]|<startport-endport>/[<proto>])
+	//        - exited=<int> containers with exit code of <int>
+	//        - health=(starting|healthy|unhealthy|none)
+	//        - id=<ID> a container's ID
+	//        - isolation=(default|process|hyperv) (Windows daemon only)
+	//        - is-task=(true|false)
+	//        - label=key or label="key=value" of a container label
+	//        - name=<name> a container's name
+	//        - network=(<network id> or <network name>)
+	//        - publish=(<port>[/<proto>]|<startport-endport>/[<proto>])
+	//        - since=(<container id> or <container name>)
+	//        - status=(created|restarting|running|removing|paused|exited|dead)
+	//        - volume=(<volume name> or <mount point destination>)
 	// produces:
 	// - application/json
 	// responses:
 	//   '200':
-	//     schema:
-	//     type: array
-	//     items:
-	//       "$ref": "#/responses/Container"
+	//        "$ref": "#/responses/DocsListContainer"
```

Comment:
`DocsListContainer` ... can we introduce a consistent naming scheme? I guess that's something for a future PR but a consistent naming scheme/coding style is invaluable for reading and understanding the code.